src/Form/index.tsx

```diff
@@ -1484,35 +1529,40 @@ function Form({
     runUserCallback(onFormComplete).then(redirectForm);
   }, [anyFinished]);
 
+  const wholePageLoader = stepLoader && (
+    <div
+      style={{
+        // FFF works for us, but not necessarily for others. thoughts?
+        // activeStep isn't set in the case of waiting on auth handshake
+        backgroundColor: `#${activeStep?.default_background_color ?? 'FFF'}`,
```

Comment:
activeStep isn't set in the case of waiting on auth handshake

FFF works for us, but not necessarily for others. thoughts?